lib/Sheet.js

```diff
@@ -84,6 +85,8 @@ class Sheet {
                 return this.cell(cellAddress);
             })
             .case(['number', '*'], (rowNumber, columnNameOrNumber) => {
+                indexValidation.validateIndex(rowNumber, "row");
```

Comment:
This is unnecessary since Sheet.cell will validate

Reply:
I have now removed this unnecessary validation.